### PR TITLE
Update select-types-of-rules-to-create.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#5462

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ You can set several rule options within a WDAC policy. Table 1 describes each ru
 | **15 Enabled:Invalidate EAs on Reboot** | When the Intelligent Security Graph option (14) is used, WDAC sets an extended file attribute that indicates that the file was authorized to run. This option will cause WDAC to periodically re-validate the reputation for files that were authorized by the ISG.| 
 | **16 Enabled:Update Policy No Reboot** | Use this option to allow future WDAC policy updates to apply without requiring a system reboot. |
 | **17 Enabled:Allow Supplemental Policies** | Use this option on a base policy to allow supplemental policies to expand it. |
-| **18 Disabled:Runtime FilePath Rule Protection** | Disable default FilePath rule protection of enforcing user-writeability and only allowing admin-writeable locations. |
+| **18 Disabled:Runtime FilePath Rule Protection** | Disable default FilePath rule protection (apps and executables allowed based on file path rules must come from a file path that’s only writable by administrator) for the path pecified in the FilePathRule parameter of the New-CIPolicyRule cmdlet. |
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md]:
```suggestion
| **18 Disabled:Runtime FilePath Rule Protection** | Disable default FilePath rule protection (apps and executables allowed based on file path rules must come from a file path that’s only writable by an administrator) for the path specified in the FilePathRule parameter of the New-CIPolicyRule cmdlet. |
```

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by illfated** [comment]
#sign-off

**Comment by mypil** [comment]
@jsuther1974 @Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5462

Thank you.

cc: @kenwith

**Pull request merged**